roles/engineblock/defaults/main.yml

```diff
@@ -51,6 +51,9 @@ engine_api_profile_password: secret
 
 engine_engine_idp_baseurlpath: "https://{{ engine_domain }}/simplesaml/"
 
+# Minimum execution time in milliseconds when response is invalid
+engine_minimum_execution_time_on_invalid_response: 5000
```

Comment:
yes I know :wink:

Would prefer `...in_invalid_received_reponse` - the received response is deemed invalid by EB, it does not have to be invalid per se...